backend/packages/Upgrade/src/api/controllers/ExperimentController.ts

```diff
@@ -1270,7 +1270,7 @@ export class ExperimentController {
     @CurrentUser() currentUser: UserDTO,
     @Req() request: AppRequest
   ): Promise<ValidatedExperimentError[]> {
-    return this.experimentService.importExperiment(experiments, currentUser, request.logger);
+    return this.moocletExperimentService.syncImportExperiment(experiments, currentUser, request.logger);
```

Comment:
`moocletExperimentService` is the entrypoint for all imports? That feels a little funny, is there something that makes `experimentService.importExperiment` unable to accommodate delegating control to `moocletExperimentService` for mooclet imports only, while remaining the same for existing imports?

Reply:
I wasn't able to import the `MoocletExperimentService` into the `ExperimentService`, since the former 'extends' the latter. Any ideas about a delegation pattern?

Reply:
We'll have to follow the same pattern with exports because we have to call `moocletExperimentService.getPolicyParametersByExperimentId()`, which we can't do from `experimentService`

Reply:
i didn't get a chance to look at this today! I feel like I ran into something similar at some point, I will take a look tomorrow to see if there's a straightforward alternative.